generator/generator_tool/generator_tool.cpp

```diff
@@ -82,6 +83,7 @@ DEFINE_uint64(planet_version, my::SecondsSinceEpoch(),
 DEFINE_string(srtm_path, "",
               "Path to srtm directory. If it is set, generates section with altitude information "
               "about roads.");
+DEFINE_bool(generate_routing_index, false, "Generates section with routing index");
```

Comment:
I suggest using "generate_routing" for brevity. 
1. The section name is routing as we agreed before;
2. We'll have only two section for routing. One for in-mwm routing and another one for cross mwm routing.